tools/openshift_qe_slack.rb

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+require_relative 'common/load_path'
+
+require 'common'
+require 'slack-ruby-client'
+
+module BushSlicer
+  class CoreosSlack
```

Comment:
Is this a class that will only work with coreos slack or it can work with other slack orgs? Name suggests it is hardcoded to coreos.

Reply:
@akostadinov the slack token is tied to the workspace which we are under `CoreosSlack`

Reply:
but token is in configuration repo, no? So with another config class can be used for other orgs.

Reply:
@akostadinov I think so, but I have not tried it with another org.

Reply:
@akostadinov now that I think about it more, this is tied to CoreosSlack org only....each app needs approval from the workspace's admin, so I expect the token is useless outside of CoreOS Slack workspace.  So yeah, this should be hardcoded name-wise.

Reply:
Well, you can create a new token and approve it and put in config to work with another org. There is nothing in code to check this is coreos slack. In other words code is agnostic to organization, it only cares to use whatever token it is provided with.

Reply:
@akostadinov I generally don't mess with the token much...but in order for the app to work in another Org outside of the `CoreOS Slack`, the admin of the new org must approve the app first I believe...again, I have not try to use this outside of CoreOS workspace.  Are you suggesting to change the name of class to `Slack` instead?

Reply:
@akostadinov tokens are associated to the app which is tied to a workspace...so I think the class name should be specific.

Reply:
If token is in config and not in code, then the line of thought that this is an org specific class is at the very least unexpected.